sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -29,6 +30,23 @@ export type AuthorizationRule = {
     secondaryKey?: string;
 };
 
+// Warning: (ae-forgotten-export) The symbol "SubscriptionRuleManagement" needs to be exported by the entry point index.d.ts
+//
+// @public (undocumented)
+export type ClientTypeT<ReceiveModeT extends "peekLock" | "receiveAndDelete", EntityTypeT extends "queue" | "subscription", SessionsEnabledT extends "sessions" | "nosessions"> = SessionsEnabledT extends "nosessions" ? EntityTypeT extends "queue" ? NonSessionReceiverTrack2<ReceiveModeT> : NonSessionReceiverTrack2<ReceiveModeT> & SubscriptionRuleManagement : EntityTypeT extends "queue" ? SessionReceiverTrack2<ReceiveModeT> : SessionReceiverTrack2<ReceiveModeT> & SubscriptionRuleManagement;
```

Comment:
Do we have to call these "Track2"? That's a bit odd for me.

Reply:
Will fix - that was just a temporary thing as I was renaming things.

Reply:
"Fixed" - I was aliasing them in index.ts for a bit but we don't have to do that now.